extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -55,6 +56,12 @@ export class BaseSlides extends BaseCarousel {
     this.shouldAutoplay_ = this.hasAutoplay_ && this.isLoopingEligible();
 
     if (this.shouldAutoplay_) {
+      this.registerAction('toggleAutoPlay', invocation => {
```

Comment:
`toggleAutoplay` to match the casing of the attribute.

Reply:
Done